src/services/completions.ts

```diff
@@ -30,7 +30,8 @@ namespace ts.Completions {
         allSourceFiles: ReadonlyArray<SourceFile>,
     ): CompletionInfo | undefined {
         if (isInReferenceComment(sourceFile, position)) {
-            return PathCompletions.getTripleSlashReferenceCompletion(sourceFile, position, compilerOptions, host);
+            const entries = PathCompletions.getTripleSlashReferenceCompletion(sourceFile, position, compilerOptions, host);

```

Comment:
not sure why is this better than what was there before?

Reply:
Makes it so `PathCompletions` do less work related to `CompletionInfo` so it can focus on just getting the paths.